client/src/www/views/contact_view/index.spec.ts

```diff
@@ -34,9 +36,12 @@ describe('ContactView', () => {
       'SentryErrorReporter',
       Object.getOwnPropertyNames(SentryErrorReporter.prototype)
     );
-    el = await fixture(
-      html` <contact-view .localize=${localize} .errorReporter=${mockErrorReporter}></contact-view> `
-    );
+    el = await fixture(html`
```

Comment:
You can use the if-message component to not block on localization.

Reply:
Yeah I know. The way we show options in the menu here doesn't lend itself so well to the `if-message` component, so I'm okay waiting on translations for this one instead of adding some complexity to handle it.